kuma/search/models.py

```diff
@@ -128,6 +127,8 @@ def promote(self):
             index_objects.delay(mapping_type, [instance.id])
             rescheduled.append(label)
         self.outdated_objects.all().delete()
+        # Allow only a single index to be promoted.
+        Index.objects.all().update(promoted=False)
```

Comment:
Ah, good point, this is a change for a race condition since `get_current()` is used often